kytos/core/kytosd.py

```diff
@@ -102,12 +104,34 @@ def main():
 
     config = KytosConfig().options['daemon']
 
+    # Configure to log uncaught exceptions to errlog file
+    sys.excepthook = exhandler
+
     if config.foreground:
         async_main(config)
     else:
         with daemon.DaemonContext():
             async_main(config)
 
+# pylint: disable=invalid-name
+def exhandler(exctype, value, tb):
+    """Define exception hook hanndler
+        Args:
+            exctype: exception type
+            value: value of exception
+            tb: traceback
+    """
+    #
+    # logs uncaught exceptions into the console and errlog.log
+    traceback.print_exception(exctype, value, tb)
+    print(tb)
+    # pylint: disable=logging-format-interpolation
+    logging.basicConfig(filename='kytos/kytos/core/errlog.log',
+                        format='%(asctime)s:%(pathname)s:'
+                        '%(levelname)s:%(message)s')
+    logging.exception('Uncaught Exception: {0}'.format(str(value)))
+    print('Uncaught Exception: {0}'.format(str(value)))
```

Comment:
Try to use f-strings instead format:
```suggestion
    print(f'Uncaught Exception: {str(value)}')
```